man/org.freedesktop.home1.xml

```diff
@@ -65,30 +65,30 @@ node /org/freedesktop/home1 {
                          out o bus_path);
       ListHomes(out a(susussso) home_areas);
       ActivateHome(in  s user_name,
-                   in  s user_record);
+                   in  s secret);
```

Comment:
Yeah, I wasn't sure how to indicate that is it an actual record, not just a string that contains the secret. Normally this would be expressed by the type of the variable, but here we effectively have a type-less system with everything masquerading as "s". The joys of untyped variables ;)